numpy/core/numeric.py

```diff
@@ -692,17 +692,24 @@ def correlate(a, v, mode='valid'):
 
     Notes
     -----
-    The definition of correlation above is not unique and sometimes correlation
-    may be defined differently. Another common definition is:
+    The definition of correlation above is used to implement 
+    the correlate function. This definition is not unique and 
+    sometimes correlation may be defined differently. 
+    Another common definition is :
 
     .. math:: c'_k = \sum_n a_{n} \cdot \overline{v_{n+k}}
 
-    which is related to :math:`c_k` by :math:`c'_k = c_{-k}`.
+    :math:`c'_k` is mathematically equivalent and computationally equivalent to 
+    :math:`c_{-k}` by :math:`c'_k = c_{-k}`.
 
     `numpy.correlate` may perform slowly in large arrays (i.e. n = 1e5) because it does
     not use the FFT to compute the convolution; in that case, `scipy.signal.correlate` might
     be preferable.
 
+    References
+    ----------
+    Wikipedia, "Cross-correlation",
+    https://en.wikipedia.org/wiki/Cross-correlation
```

Comment:
Per [the numpydoc standard](https://numpydoc.readthedocs.io/en/latest/format.html#references), the `References` section is intended for enumerating reference targets for in-line references in the docstring (i.e. there should be a `.. [1]` and corresponding `[1]_` in the text).